web/explorer/src/assets/main.css

```diff
@@ -0,0 +1,20 @@
+body {
+    margin: 0 auto;
+    font-weight: normal;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+a {
+    text-decoration: none;
```

Comment:
any strong feelings on showing links underlined so it's easier to see them?
if you think it's good as is, let's leave as is for now

Reply:
I believe displaying the default underline for all the links will overwhelm the page.